dymos/trajectory/test/test_trajectory_input_parameters.py

```diff
@@ -27,8 +27,8 @@ def make_traj(transcription='gauss-lobatto', transcription_order=3, compressed=F
         traj.add_parameter('c', opt=False, val=1.5, units='DU/TU',
                            targets={'burn1': ['c'], 'burn2': ['c']})
     elif param_mode == 'param_sequence_missing_phase_deprecated':
-        traj.add_input_parameter('c', val=1.5, units='DU/TU',
```

Comment:
I think you can remove this branch of the if statement, plus the test `test_input_parameter_deprecated`, as that stuff is just there to make sure the deprecated `add_input_parameter` still works.

Reply:
fixed and pushed